moderne-platform-adventure/README.md

```diff
@@ -1,98 +1,113 @@
+   recipe will run against `1` repository. If you want more details about the
+   recipe, you can click on the `More Details` link. From there, you can find
+   additional information such as a link to the source code for this recipe (all
+   of the recipes are OpenRewrite recipes).
 
-5. Click on `DRY RUN`.
+5. To begin running the recipe, click on the `DRY RUN` button.
 
-6. Then you will see a recipe running, and when it is completed, you can click on the repository
-name to see the results. 
+6. You will now be redirected to a page that shows all of the relevant recipe
+   run information. Once it's finished, you can click on the repository name to
+   see the results:
 
 ![results](assets/execution.png)
 
-7. It is important to see that the results contain:
-
-  - The removal of `@Autowired`.
-  - The replacement of Junit 4 for JUnit 5.
-  - The replacement of `javax` for `jakarta`.
-  - It is migrated to Java 17 and uses text blocks.
-  - Some best practices are applied such as `public` test method modifiers.
- 
-
-## Security Vulnerabilities using the Moderne Platform
-
-Another use case to use the platform is for detecting CVEs in the dependencies
-and upgrade a set of repositories with the last versions. Since the Moderne platform
-supports complex refactorings such as the SpringBoot 3 migration, it is expected
-that the community can contribute and provide recipe for other major migrations that
-not only require to bump the dependency version. In this challenge, we are going to see
-how we can see the list of vulnerabilities in important open source repositories and
-create a composite refactor with many the minor vulnerabilities that we want to fix.
-
-1. In this case, we are going to use the `Check for dependency vulnerabilities` recipe.
-This recipe offers three options. For demo purposes, we recommend to simply 
-select:
-
-    - Scope:  `compile`. 
-    - Overwrite management version: true.
-    - Add markers: No value selected.
-
-2. In the top menu, select the default repository group, which is composed by a hundred (142) of
-open source Netflix and Spring repositories.
-
-3. Select `DRY RUN`, this should take 10 minutes approx. Now, clicking on each of the repositories, 
-you should be able to see what dependency changes are required to remove vulnerabilities, in 
-case dependencies can be locally fixed.
-
-4. Click on data tables (right side blue button) and download the vulnerability report in CSV format.
-
-5. Open the CSV with Google spreadsheets or your preferred tool.You will see that the CSV contains
-an entry per vulnerability, depedency and repository. There is an interesting column called
-`fixedVersion`, which tells you what is the immediate version that fixes the vulnerability. It
-is also important to notice that there is a `depth` column because we also analyze transitive
-dependencies.
-
+7. If you look at the results you should see that:
```

Comment:
@timtebeek I believe that this is the summary you want to make it explicit in the maven and gradle adventure as well